pkg/apis/enterprise/v1alpha3/clustermaster_types.go

```diff
@@ -31,8 +31,11 @@ import (
 type ClusterMasterSpec struct {
 	CommonSplunkSpec `json:",inline"`
 
-	// Splunk Smartstore configuration. Refer to indexes.conf.spec on docs.splunk.com
+	// Splunk Smartstore configuration. Refer to indexes.conf.spec and server.conf.spec on docs.splunk.com
 	SmartStore SmartStoreSpec `json:"smartstore,omitempty"`
+
+	// Image to use for Spark pod containers (overrides RELATED_IMAGE_SPLUNK_SPARK environment variables)
```

Comment:
As a comment please explain why we are using SparkImage within CM (initContainer)